src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1592,18 +1600,50 @@ protected override void VisitArguments(ImmutableArray<BoundExpression> arguments
             throw ExceptionUtilities.Unreachable;
         }
 
-        private void VisitArguments(ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> refKindsOpt, MethodSymbol method, ImmutableArray<int> argsToParamsOpt, bool expanded)
+        private void VisitArguments(
```

Comment:
Just to double-check: this overload (for methods) and the following one (for properties) are used?

Reply:
Yes, each overload has a couple of callers.

---
In reply to: [182817432](https://github.com/dotnet/roslyn/pull/26107#discussion_r182817432) [](ancestors = 182817432)